.github/workflows/build-base-images.yml

```diff
@@ -36,7 +36,9 @@ jobs:
         language: [python, cpp]
 
     name: "Building image"
-    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.get-tag.outputs.tag }}
+    runs-on: ubuntu-22.04
```

Comment:
suggestion: create an env to use throughout the file

Reply:
I guess env won't work, because the jobs are executed by different agents and envs are only saved on one agent. Or do you mean something else?

Reply:
`on:
  schedule:
    - cron: '0 5 * * 0'
  workflow_dispatch:
  push:
    # Run only on branches/commits and not tags
    branches:
      - main
  pull_request:
    branches:
      - main

env:
  UBUNTU_VERSION: ubuntu-22.04
`

`
runs-on: $UBUNTU_VERSION
`

Reply:
ah ok I thought you mean instead of the output.. 
I will change this.

Reply:
Reverted, can't be used due to: https://github.com/actions/runner/issues/480